### DISCUSSION TITLE
About License

### DISCUSSION THREAD

**Discussion by hwpplayer1** [arendst/Tasmota#18157] [Q&A] [answered]
Could you please clarify if your project's license GPL3 or GPL3.0+ ?

Do you mind to add the or later spelled out in the README file.

**Comment by s-hadinger** [comment, marked as answer]
The license is here : https://github.com/arendst/Tasmota/blob/development/LICENSE.txt

**Comment by willtop43** [comment]
Thank you for the link to the license. However, the license by itself does not tell us if it is GPLv3-or-later or GPLv3-only. Judging from many of the code file's headers, it seems to be GPLv3-or-later, but having this declared in the README file would make it much easier to determine the -or-later choice. The README file already has a license section and that would be the optimal place to include the -or-later information. Thank you for your consideration.

**Comment by arendst** [comment]
It's GPL-3.0-only
